test/support/uffizzi_preview_stub_support.rb

```diff
@@ -5,34 +5,46 @@
 module UffizziPreviewStubSupport
   include ApiRoutes
 
-  def stub_uffizzi_preview_list(status, body, project_slug)
+  def stub_uffizzi_preview_list(body, project_slug)
     url = deployments_uri(Uffizzi.configuration.server, project_slug)
 
-    stub_request(:get, url).to_return(status: status, body: body.to_json)
+    stub_request(:get, url).to_return(status: 200, body: body.to_json)
```

Comment:
if this stub always return with status 200 may be we should change name to `stub_uffizzi_preview_list_success`